libs/shared/babel/src/web.ts

```diff
@@ -0,0 +1,32 @@
+import transformLib from './transformLib'
+import { TransformOptions, ConfigAPI } from '@babel/core'
+
+export default function (api: ConfigAPI): TransformOptions {
+  const config = {
+    plugins: [],
+  } as TransformOptions
+
+  if (api.env('production') && config.plugins) {
+    config.plugins.push([
+      'transform-imports',
+      {
+        ...transformLib('@island.is/island-ui/core'),
+        ...transformLib('@island.is/island-ui/contentful'),
+        ...transformLib(
+          '@island.is/web/components',
+          'apps/web/components/index.ts',
+        ),
+        lodash: {
+          transform: 'lodash/${member}',
+          preventFullImport: true,
+        },
+        'date-fns': {
+          transform: 'date-fns/${member}',
+          preventFullImport: true,
```

Comment:
`lodash` and `date-fns` are already being handled by `eslint` via the `no-restricted-imports` rule

... but doesn't hurt :)

Reply:
Ah, cool!